### ISSUE TITLE
Scroll bars and Project title not following the theme on windows

### ISSUE THREAD

**Issue by cyanotrix** [pineapplegiant/spaceduck#53]
Not sure if others have faced it but on windows 3 elements were out of whack, or maybe cause I'm not using vim the scrollbars were ugly. Anyway I have attached the before and after fix screenshots along with the fix. Let me know if you can reproduce the issue, you can use the style to fix it.

Issue
![atom-spaceduck-ui bug](https://user-images.githubusercontent.com/5426801/124989633-f3fd2a00-e05c-11eb-8e1b-b21411a20916.png)

Fixed
![atom-spaceduck-ui fix](https://user-images.githubusercontent.com/5426801/124989680-037c7300-e05d-11eb-9f55-36d041c2e003.png)

**Comment by pineapplegiant** [comment]
Oh nice, thanks for opening an issue. That does look better. We should move this issue to the atom port. I’m still working on moving all the things to their right place so sorry about that being confusing.

**Comment by cyanotrix** [comment]
No worries, happy to help. Love the theme, after a long time found something that I can work with.

**Comment by pineapplegiant** [comment]
That’s awesome to hear. 
Yeah hopefully gonna release some final changes to the palette and standardize everything. Color schemes are hard lemme tell you

**Comment by cyanotrix** [comment]
Yeah no kidding, I'm experimenting on your theme with comments colors, there are not super contrasty compared to the background and hard to read in dimly lit env. I'll post some screenshots in case something looks good in line with the palette.

**Comment by pineapplegiant** [comment]
Yeah they’re pretty bad lol. 
We have the dev branch with new comment colors. I’m gonna include that in the next release don’t worry. I was using the iTerm terminal emulator when I first created the theme, and had a setting that increased the contrast so the colors were all off when I started testing it in different places.
